publication/ver11/societal_impact.md

```diff
@@ -0,0 +1,216 @@
+together.
+
+The WoT standards can be considered "essential" when this metadata is not available
+by other means or would be expensive or inconvenient to obtain, or where machine-readable
+metadata is needed to automate integration.
+
+When combined with a browser, the consistent metadata provided by WoT standards
+can be used to automatically build web user interfaces for IoT devices, making them
+more consistently accessible.  This may or may not be done by directly talking to 
+IoT devices.  Very often an IoT system will digest the metadata about a set of
+IoT devices and use this to generate a consistent interface, such as a dashboard.
+
+Some usages of WoT may be safety-critical, for example, an IoT interface to a factory
+actuator or to a fall-detection system in the home.  A government may also use WoT
+to provide access to public services, including critical mechanisms such as access
+to environmental monitoring systems (which might in turn be used to warn vulnerable
+people about air pollution states they need to be wary of). 
+
+The WoT standards primarily describe metadata and how to access it.  To mitigate
+failures, such as the inability to access metadata if a WoT Discovery service
+is unavailable, such systems should use WoT Discovery and metadata (WoT Thing
+Descriptions) for initial connections and if updated metadata is not available,
+use the last known state of this metadata.
+
+### 2. [What kinds of activities could your specification become a part of that you are not designing for?](https://w3ctag.github.io/societal-impact-questionnaire/#unexpected-use)
+WoT does not directly address accessibility but IoT systems generally can be
+used to build custom user interfaces, including those that make systems easier
+to use (e.g. large buttons to replace difficult to use controls) or map sensory
+modalities (e.g. mapping visual indicators to audible ones or vice-versa).
+Unfortunately, the current WoT specification describes only network behavior,
+not physical behavior, making automated construction of such interfaces difficult.
+However, since WoT uses JSON-LD, it would be possible in theory to annotate WoT
+Thing Descriptions with information such as this.
+
+In general, additional metadata can be attached to WoT Thing Descriptions and
+this can be used for arbitrary purposes that can go beyond what is currently
+in the specification.  For example, a given device manufacturer could attach
+geolocation information to a WoT Thing Description.  To mitigate privacy
+hazards from both current and future metadata in WoT Thing Descriptions, the 
+WoT Discovery process provides mechanisms to control access to WoT Thing Descriptions.
+
+### 3. [What risks do you see in features of your specification being misused, or used differently from how you intended?](https://w3ctag.github.io/societal-impact-questionnaire/#misuse)
+Many possible misuses and risks are already discussed in the answers to the Security and
```

Comment:
we should provide a link to this questionaire

Reply:
Every question already links to the associated section of the questionnaire.  There is also already a link at the top of the document.  But re-reading this I guess you meant by "this" the *security* questionnaire.  Sure, that makes sense... it was actually already linked in another question, but I added a link here too.